Sources/Workspace/Workspace+Registry.swift

```diff
@@ -257,18 +257,29 @@ extension Workspace {
                     var modifiedDependencies = [TargetDescription.Dependency]()
                     for dependency in target.dependencies {
                         var modifiedDependency = dependency
-                        if case .product(let name, let packageName, let moduleAliases, let condition) = dependency,
-                           let packageName
-                        {
-                            // makes sure we use the updated package name for target based dependencies
-                            if let modifiedPackageName = targetDependencyPackageNameTransformations[packageName] {
+                        switch dependency {
+                        case let .product(name: name, package: packageName, moduleAliases: moduleAliases, condition: condition):
+                            if let packageName,
+                               // makes sure we use the updated package name for target based dependencies
+                               let modifiedPackageName = targetDependencyPackageNameTransformations[packageName] {
                                 modifiedDependency = .product(
                                     name: name,
                                     package: modifiedPackageName,
                                     moduleAliases: moduleAliases,
                                     condition: condition
                                 )
                             }
+                        case let .byName(name: name, condition: condition):
```

Comment:
Can you provide a more verbose name to the bound `name` constant? This needs to be clearer on which name it's referring to, similarly to the case handled above.

Reply:
I changed it to `packageName`. Let me know if you'd like to change it to something else 🙏